librexgen.patch

```diff
@@ -0,0 +1,21 @@
+From 9d1c30baf94b2fdcde6906c9f449109133f38a0b Mon Sep 17 00:00:00 2001
```

Comment:
librexgen has been removed from the upstream (ok, it has been reactivated at https://github.com/openwall/john/issues/4640). Anyway, I intend to remove it in 1.9.0-Jumbo-2 flatpak version.